edgelet/edgelet-core/src/settings.rs

```diff
@@ -49,6 +77,35 @@ impl Listen {
     }
 }
 
+impl Default for Listen {
+    // Clippy wants us to use `option_env!("...").unwrap_or("...")` but that can't be used in consts.
+    #[allow(clippy::option_if_let_else)]
+    fn default() -> Self {
+        const DEFAULT_MANAGEMENT_URI: &str =
+            if let Some(value) = option_env!("IOTEDGE_CONNECT_MANAGEMENT_URI") {
+                value
+            } else {
+                "fd://aziot-edged.mgmt.socket"
+            };
+        const DEFAULT_WORKLOAD_URI: &str =
+            if let Some(value) = option_env!("IOTEDGE_CONNECT_MANAGEMENT_URI") {
```

Comment:
This should be `IOTEDGE_CONNECT_WORKLOAD_URI`.

Reply:
Fixed.